examples/models/llama/export_llama_args.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Why do we have to have `export_llama_args.py` and `export_llama_hydra.py`? Can we merge them into `export_llama.py`?